### PR TITLE
update van_der_pol exercise

### PR DESCRIPTION
**Pull request by meenalparakh**
This PR updates the notebook and the corresponding test file.

- - -
This change is [Reviewable](https://reviewable.io/reviews/RussTedrake/underactuated/508)

### CODE REVIEW

File: exercises/lyapunov/van_der_pol.ipynb [hunk 1]
```diff
@@ -90,22 +106,32 @@
...
+    "deepnote_cell_type": "markdown"
+   },
    "source": [
     "## Lyapunov Function via Linearization\n",
     "The approach we will follow to estimate the ROA of the oscillator is the following:\n",
     "\n",
     "- We linearize the dynamics $\\dot{\\mathbf{x}} = f(\\mathbf{x})$ around the origin, to get $\\dot{\\mathbf{x}} = A \\mathbf{x}$.\n",
     "\n",
-    "- We solve the Lyapunov equation $$A^T P + P A = - Q$$ to get the matrix $P$, and the Lyapunov function $V(\\mathbf{x}) = \\mathbf{x}^T P \\mathbf{x}$ for the linearized system.\n",
+    "- We solve the Lyapunov equation $$A^T P + P A = - Q$$ to get the positive definite matrix $P$, and the Lyapunov function $V(\\mathbf{x}) = \\mathbf{x}^T P \\mathbf{x}$ for the linearized system. **Note**: A standard choice for Q is an identity matrix.\n",
```
Comment (RussTedrake) [inline exercises/lyapunov/van_der_pol.ipynb:134]: "Q" => "$Q$"

File: exercises/lyapunov/van_der_pol.ipynb [hunk 2]
```diff
@@ -114,40 +140,50 @@
     "In words, we try to find the largest level set $L(\\rho)$ entirely contained the region of space where $\\dot{V}(\\mathbf{x})$ is negative.\n",
     "Lyapunov theory tells us that any trajectory that starts inside such a set will eventually converge to the origin.\n",
     "\n",
-    "We start by deriving $V(\\mathbf{x})$."
+    "We start by deriving $V(\\mathbf{x})$. In the next cell, write down the matrix A (obtained by linearizing $f(x)$ near the origin), the positive definite matrix Q, and compute P using the drake function `RealContinuousLyapunovEquation`"
```
Comment (RussTedrake) [inline exercises/lyapunov/van_der_pol.ipynb:143]: put A, Q, P in math mode with $ $

File: exercises/lyapunov/van_der_pol.ipynb [hunk 3]
```diff
@@ -540,31 +610,57 @@
...
+    "Created in <span style='font-weight:600;margin-left:4px;'>Deepnote</span></a>"
```
Comment (RussTedrake) [inline exercises/lyapunov/van_der_pol.ipynb:651]: definitely shouldn't be including the deepnote image in the metadata!

**Review by RussTedrake — CHANGES_REQUESTED** [review]

File: exercises/lyapunov/van_der_pol.ipynb (hunk 3 above)
Comment (meenalparakh) [reply exercises/lyapunov/van_der_pol.ipynb]: Changed to math mode and deleted the deepnote cell in the notebook.

**Comment by RussTedrake** [comment]
i've merged the two problems directly.

**Pull request closed without merging**